scripts/cmake/ElektraCache.cmake

```diff
@@ -135,7 +135,7 @@ set (
 
 set (
 	KDB_DB_DIR
-	".dir"
+	".dir" # TODO (kodebach) [Q]: wouldn't .elektra be better? similar to .git
```

Comment:
Please remove such TODO which are not TODO and create proposals.

Reply:
Will do, I'll also collect them in an issue so we don't loose the ideas